source/features/copy-file.tsx

```diff
@@ -17,8 +17,19 @@ async function init(): Promise<void> {
 	// This won't show button on markdown and binary files until the code is loaded
 	if (select.exists('.blob.instapaper_body')) {
 		delegate('.rgh-md-source', 'rgh:view-markdown-source', () => {
+			// Prevents several buttons
+			if (select('[aria-label="Copy file to clipboard"]')) {
+				return
+			}
 			renderButton();
 		});
+		delegate('.rgh-md-source', 'rgh:view-markdown-rendered', () => {
+			// Prevents several buttons
+			const buttonSelector = '[aria-label="Copy file to clipboard"]'
+			if (select.exists(buttonSelector)){
+				select(buttonSelector)!.remove()
+			}
+		});
```

Comment:
I think actually we only need this check here, because if we remove it every time it won't be there afterwards.

```js

		delegate('.rgh-md-source', 'rgh:view-markdown-source', renderButton);
		delegate('.rgh-md-source', 'rgh:view-markdown-rendered', () => {
			const button = select('.rgh-copy-file');
			if (button){
				button.remove()
			}
		});
```

I noticed we have a `copy-btn` class on the button, we should rename it to `rgh-copy-file`.

Also